src/debugging/netcore/NetCoreDebugHelper.ts

```diff
@@ -278,19 +287,17 @@ export class NetCoreDebugHelper implements DebugHelper {
         return pathNormalize(result, platformOS);
     }
 
-    private async installDebuggerInContainer(containerName: string): Promise<string> {
+    private async installDebuggerInContainer(containerName: string, installPath: string): Promise<string> {
         const yesItem: MessageItem = DialogResponses.yes;
         const install = (yesItem === await window.showInformationMessage(localize('vscode-docker.debug.netcore.attachingRequiresDebugger', 'Attaching to container requires .NET Core debugger in the container. Do you want to install debugger in the container?'), ...[DialogResponses.yes, DialogResponses.no]));
         if (!install) {
-            throw new UserCancelledError();

```

Comment:
The message from UserCancelledError will never be shown; which is why I'd removed the string when doing the loc work. Is it needed?

Reply:
Nit if we keep it: single quotes, wrap in localize call